site/styles/_variables.scss

```
@@ -35,8 +35,8 @@ $container-max-widths: (
 
 $primary:         #35a0a8 !default;
 $secondary:       #4cd3dd !default;
-$body-bg:         #fff !default;
-$body-color:      #000 !default;
+$body-bg:         #000000 !default;
+$body-color:      #ffffff !default;
```

Nit: prefer the short option when available.

```suggestion
$body-bg:         #000 !default;
$body-color:      #fff !default;
```

---

done